### PR TITLE
Issue #7361; feature info selector fixed

### PR DESCRIPTION
**Pull request by tonym3nsa**
## Description

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [X] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#7361 

**What is the new behavior?**

Selected feature info card is highlight and there is no error in console
## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/components/TOC/TOCItemsSettings.jsx [hunk 1]
```diff
@@ -123,7 +124,7 @@ const TOCItemSettings = (props) => {
                         nodeType={settings.nodeType}
                         settings={settings}
                         retrieveLayerData={onRetrieveLayerData}
-                        onChange={(key, event) => isObject(key) ? onUpdateParams(key, realtimeUpdate) : onUpdateParams({[key]: event.target.value, label: event.target.label}, realtimeUpdate)}
```
Comment (tonym3nsa) [inline web/client/components/TOC/TOCItemsSettings.jsx:126]: The `onChange` event is passing an object called `event`. The function handled by the event was expecting the event object to having keys and values for `event.target.value `and `event.target.label` but those seem to be non-existent; the added fix handles the function the absence of the aforementioned keys.

File: web/client/utils/TOCUtils.js [hunk 2]
```diff
@@ -115,3 +115,13 @@ export const getLabelName = (groupLabel = "", groups = []) => {
     label = label.replace(/\${dot}/g, '.');
     return label;
 };
+
+/**
+ * Returns value of selected settings card
+ * @param {(string|object)} event
+ * @param {string=} key
```
Comment (MV88) [inline web/client/utils/TOCUtils.js:122]:
```suggestion
 * @param {string=} key, default is 'value'
```

**Review by MV88 — APPROVED** [review]
LGTM

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

@tonym3nsa please prepare backport 2021.02.xx as usual

**Comment by tdipisa** [comment]
> @ElenaGallo please test in DEV
> 
> @tonym3nsa please prepare backport 2021.02.xx as usual

@MV88 we have to ask for a backport only if the test in DEV is passed, so please just ask @ElenaGallo for a test in DEV first then she will let @tonym3nsa know for the backport as usual.

**Comment by ElenaGallo** [comment]
Test passed, @tonym3nsa please backport to 2021.02.xx. Thank you

**Pull request merged**